src/t_zset.c

```diff
@@ -2184,6 +2188,165 @@ inline static void zunionInterAggregate(double *target, double val, int aggregat
     }
 }
 
+int dictGetMaxElementLength(dict *d) {
+    dictIterator *di;
+    dictEntry *de;
+    size_t maxelelen = 0;
+
+    di = dictGetIterator(d);
+
+    while((de = dictNext(di)) != NULL) {
+        sds ele = dictGetKey(de);
+        if (sdslen(ele) > maxelelen) maxelelen = sdslen(ele);
+    }
+
+    return maxelelen;
```

Comment:
i see you forgot to release the dict iterator (memory leak).

Reply:
Fixed.